Unity-Technologies/ScriptableRenderPipeline
# [Backports] Shader Graph FogBugz Fixes

### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

Backports the following bug fixes:
FB#1131211 - Voronoi Node Serialization Issue PR #3138
FB#1131755 - Drag-dropping normal map texture into a shader graph causes errors PR #3137
FB#1132342 - Multiply Bug. Zeroing components in code generation PR #3134
FB#1131646 - IsFrontFace Breaks Subgraphs PR #3133

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
All Tests Green
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=sg%2Fbug-fix-backports&unity_branch=2019.1%2Fstaging&automation-tools_branch=master

**Manual Tests**: What did you do?
- Other: 
Opened project and hand-tested each fix multiple times to confirm errors are gone.

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High? Low

**Halo Effect**: None, Low, Medium, High? Low

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### DO NOT MERGE UNTIL CORRESPONDING PRS HAVE GONE INTO MASTER
